.devcontainer/devcontainer.json

```diff
@@ -86,4 +86,6 @@
       }
     }
   }
+
+  "forwardPorts": [8080]
```

Comment:
Weird, I use notebooks in the dev container often and didn't need this.

Reply:
It was needed to use `snakeviz` with it, for some reason.